### PR TITLE
Overall doc fixes v1499 dev1

### PR DESCRIPTION
**Pull request by ru-fu**
Fixes as discussed in the doc review meeting.
Some fixes (that should not go in before we are ready to tag) are added to https://github.com/nrfconnect/sdk-nrf/pull/3412 instead.

### CODE REVIEW

**Comment by peknis** [comment]
Line 31 in ug_nrf5340.rst should have DK instead of PDK, right?

**Comment by ru-fu** [comment]
> 
> 
> Line 31 in ug_nrf5340.rst should have DK instead of PDK, right?

Yes, I've changed that in https://github.com/nrfconnect/sdk-nrf/pull/3412 together with the link to the DK UG. :)

**Review by umapraseeda — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -91,12 +91,12 @@ Sending data before connecting to nRF Cloud
...
 ============
 
-.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0
+.. rst-class:: v1-4-99-dev1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
 
 NCSDK-5666: LTE Sensor Gateway
   The :ref:`lte_sensor_gateway` sample crashes when Thingy:52 is flipped.
 
-.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0
+.. rst-class:: v1-4-99-dev1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0
```
Comment (umapraseeda) [inline doc/nrf/known_issues.rst:99]: This is fixed, as informed by @nordic-hani 
```suggestion
.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0
```

Comment (ru-fu) [reply doc/nrf/known_issues.rst]: Updated.

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -882,17 +882,17 @@ tx_buffer_length set incorrectly
...
 ******
 
-.. rst-class:: v1-4-0
+.. rst-class:: v1-4-99-dev1 v1-4-0
 
 NCSDK-6330: USB Mass Storage Sample Application fails MSC Tests from USB3CV test tool
   :ref:`zephyr:usb_mass` fails the USB3CV compliance Command Set Test from the MSC Tests suite.
 
-.. rst-class:: v1-4-0
+.. rst-class:: v1-4-99-dev1 v1-4-0
```
Comment (jfischer-no) [inline doc/nrf/known_issues.rst:890]: NCSDK-6328 is fixed upstream/sdk-zephyr.

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -882,17 +882,17 @@ tx_buffer_length set incorrectly
...
 
 NCSDK-6328: USB CDC ACM Composite Sample Application fails Chapter 9 Tests from USB3CV test tool
   :ref:`zephyr:usb_cdc-acm_composite` fails the USB3CV compliance TD 9.1: Device Descriptor Test from the Chapter 9 Test suite.
 
-.. rst-class:: v1-4-0
+.. rst-class:: v1-4-99-dev1 v1-4-0
```
Comment (jfischer-no) [inline doc/nrf/known_issues.rst:895]: NCSDK-6331 is fixed upstream/sdk-zephyr.

**Review by jfischer-no — CHANGES_REQUESTED** [review]

File: doc/nrf/known_issues.rst (hunk 2 above)
Comment (ru-fu) [reply doc/nrf/known_issues.rst:890]: Removed.

File: doc/nrf/known_issues.rst (hunk 3 above)
Comment (ru-fu) [reply doc/nrf/known_issues.rst:895]: Removed.

**Review by jfischer-no — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 4]
```diff
@@ -91,7 +91,7 @@ Sending data before connecting to nRF Cloud
 Other issues
 ============
 
-.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0
+.. rst-class:: v1-4-99-dev1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
```
Comment (nordic-hani) [inline doc/nrf/known_issues.rst:94]: LTE sensor gateway crash was fixed and should be removed from known issues after v1.4.0 🙂

**Review by nordic-hani — APPROVED** [review]

**Review by MarekPorwisz — APPROVED** [review]

**Pull request merged**